Automattic/woocommerce-payments
# Adjust texts and links in WC admin express checkout section

Fixes #4843 

#### Changes proposed in this Pull Request
A couple of changes/adjustments need to be made to the texts and links in the admin express checkout section:
<img src="https://user-images.githubusercontent.com/82826872/193993541-0aa3a6f1-aae5-4836-ab34-a92a117a6ae0.png" />
* Update the “learn more” link that links to an express checkout section in the documentation where merchants can easily find information on Apple Pay, Google Pay, and Link
* In Link subsection include a hyperlink to the[ Link docs support article](https://woocommerce.com/document/stripe/#section-11)
* In Link subsection, include copy that state the “new payment experience (UPE) needs to be enabled for Link”
* Change “Stripe Link” to “Link by Stripe”
* Add description under Apple Pay / Google Pay – “Boost sales by offering a fast, simple, and secure checkout experience”
* Under Link - add description that includes ““Link autofills your customers’ payment and shipping details to deliver an easy and seamless checkout experience. By enabling this feature, you agree to the Link by Stripe terms and privacy policy.“








#### Testing instructions





* Go to WC admin -> Settings -> Payments
* check the express checkout section if all changes above are made

-------------------


**Post merge**
